openproblems/tasks/_batch_integration/batch_integration_feature/api.py

```diff
@@ -25,25 +20,7 @@ def check_method(adata, is_baseline=False):
     return True
 
 
-@dataset()
-def sample_dataset():
-    """Create a simple dataset to use for testing methods in this task."""
-    import scanpy as sc
-
-    adata = load_sample_data()
-
-    adata.var.index = adata.var.gene_short_name.astype(str)
-    sc.pp.normalize_total(adata)
-
-    adata.obsm["X_uni"] = sc.pp.pca(adata.X)
```

Comment:
This line is missing in the common version of the api as I see it. Is this used @danielStrobl ?

Reply:
X_uni doesn't appear to be used anywhere (but X_uni_pca is)